kuma/search/tests/test_views.py

```diff
@@ -19,9 +19,9 @@ def test_search_filters(self):
         response = self.client.get('/en-US/search?q=article')
         assert response.status_code == 200
         assert_no_cache_header(response)
-        assert 'Results for' in response.content
-        assert 'an article title' in response.content
-        assert '4 documents found for "article" in English' in response.content
+        assert 'Results for' in response.content.decode('utf-8')
```

Comment:
nit: I've been using this pattern other places, which feels less repetitive and less magical to me:

```
content = response.content.decode(response.charset)
assert 'Results for' in content
assert 'an article title' in content
assert '4 documents found for "article" in English' in content
```